tests/s3/copy_object/test_copy_obj_simultaneous_del.py

```diff
@@ -137,6 +155,45 @@ def create_put_parallel_copy_and_delete_object(self, copy_args, delete_args, is_
                 process2 = pool.apply_async(self.delete_object_wrapper, args=delete_args)
                 assert_utils.assert_true(process1.get()[0], process1.get()[1])
                 assert_utils.assert_true(process2.get()[0], process2.get()[1])
+            if copy_args[0] == copy_args[2] and copy_args[1] == copy_args[3]:
+                self.head_object_wrapper(copy_args[0], copy_args[1], "404")
```

Comment:
You can pass the error message already being passed to the create_put_parallel_copy_and_delete_object in copy_args rather than passing "404" as the exception

Reply:
addressed